oboe-sys/build.rs

```diff
@@ -99,8 +99,8 @@ fn main() {
 
         if cfg!(feature = "static-link") {
             link_args.push(SharedLib("log".into()));
-            link_args.push(SharedLib("OpenSLES".into()));
         }
+        link_args.push(SharedLib("OpenSLES".into()));
```

Comment:
Why dynamic linking needs OpenSLES? As I understand correctly _liboboe-ext.so_ only needs it.

Reply:
This is what I was getting without the fix:
```
ld: warning: libOpenSLES.so, needed by PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so, not found (try using -rpath or -rpath-link)
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `slCreateEngine'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_ENGINE'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_RECORD'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_ANDROIDCONFIGURATION'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_ANDROIDSIMPLEBUFFERQUEUE'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_PLAY'
          PROJECT_PATH/target/aarch64-linux-android/debug/deps/liboboe-ext.so: undefined reference to `SL_IID_BUFFERQUEUE'
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

So apparently the linker requires to specify the dependent libraries too.

Reply:
Hmm. I have never to got so.